### ISSUE TITLE
supervisors/admins are notified when volunteer closes followup generated by them

### ISSUE THREAD

**Issue by Zrrrpy** [rubyforgood/casa#1507]
Part of epic #1495 | dependent on #1496

As a `supervisor` or `admin`,
I want to receive a notification when a `volunteer` resolves a follow up I generated on a `case_contact` associated with their `case_assignment`,
so that I know when the followup task has been completed. 

### How it works now
When #1506 is complete, any user will be able to resolve a follow up, but the only way to know that follow up has been resolved will be by remembering which `case_contact` it was associated with, and viewing that `case_contact`.

### How it should work instead.
`Supervisors` and `admins` should be notified when a `volunteer` resolves a follow up generated by them ("them" meaning the `supervisor` or `admin`).

### Acceptance Criteria
- [x] When a volunteer resolves a follow up generated by a `supervisor` or `admin`, the `supervisor` or `admin` who generated that follow up receives a notification.
- [x] That notification appears in the user's notifications inbox. 
   - [x] the inbox should be accessible via sidebar menu by clicking on the hyperlinked text: Inbox
   - [x] when there is a new or unread notification in the inbox, a simple badge should display on it in the sidebar menu, such as a red dot.
- [x] The notification should be text that reads something like this example: _Stacey resolved a follow up. Click to see more._
   - [x] _Click to see more_ should be hyperlinked text directing to the Edit page for the `case_contact` associated with the resolved follow up item.

**Comment by scottolsen** [comment]
Hoping to knock this out this weekend. :)

**Comment by scottolsen** [comment]
@Zrrrpy 
>  When a volunteer resolves a follow up generated by a supervisor or admin, the supervisor or admin who generated that follow up receives a notification.

I'm not sure if this is a valid scenario I need to code for but if a volunteer creates a followup and then a different volunteer resolves it, do I need to make sure no notification is generated?
